jerry-core/jrt/jrt.h

```diff
@@ -184,7 +181,7 @@ template<typename... values> extern void jerry_ref_unused_variables (const value
 /**
```

Comment:
In this file there's also the `jerry_ref_unused_variables` function which currently has argument name